pokete.py

```diff
@@ -1578,15 +1636,16 @@ def balls_label_rechar():
     """Rechars the balls label"""
     movemap.balls_label.text = ""
     for i in range(6):
-        movemap.balls_label.text += "-" if i >= len(figure.pokes) or figure.pokes[i].identifier == "__fallback__" else "o" if figure.pokes[i].hp > 0 else "x"
+        movemap.balls_label.text += "-" if i >= len(figure.pokes) or figure.pokes[
```

Comment:
This one looks cursed.

Reply:
Ok. See it.

Fixed in 900135ff7a571b957d878156345a9a4e82278b0f